pipes-and-filters/PipesAndFilters.Shared/ServiceBusPipeFilter.cs

```diff
@@ -37,56 +46,52 @@ public void Start()
             {
                 ServiceBusUtilities.CreateQueueIfNotExistsAsync(this.connectionString, this.outQueuePath).Wait();
 
-                this.outQueue = QueueClient.CreateFromConnectionString(this.connectionString, this.outQueuePath);
+                this.queue = new ServiceBusClient(this.connectionString);
             }
 
             // Wait for queue creations to complete
             createInQueueTask.Wait();
 
             // Create inbound and outbound queue clients
-            this.inQueue = QueueClient.CreateFromConnectionString(this.connectionString, this.inQueuePath);
+            this.queue = new ServiceBusClient(this.connectionString);
```

Comment:
The equivalent of the inQueue would be the processor. It is fine if you want to move lines 30-36 to here to avoid making changes unrelated to the migration to the new library.